src/firmware/main/app/world/BUILD

```diff
@@ -0,0 +1,109 @@
+package(default_visibility = ["//visibility:public"])
+
+# TODO: remove all `strip_include_prefix` once firmware has been setup to have includes
```

Comment:
I thought about it, but decided not to leave an issue because this is an area of such active development that if we were to be consistent an leave issues at this level of granularity, we'd probably have a 100+ more. This should be addressed in my next few PR's.